libqtile/core/manager.py

```diff
@@ -919,26 +860,26 @@ def call_soon(self, func, *args):
         def f():
             func(*args)
             self.conn.flush()
-        return self._eventloop.call_soon(f)
+        return self._event_loop.call_soon(f)
 
     def call_soon_threadsafe(self, func, *args):
         """ Another event loop proxy, see `call_soon`. """
         def f():
             func(*args)
             self.conn.flush()
-        return self._eventloop.call_soon_threadsafe(f)
+        return self._event_loop.call_soon_threadsafe(f)
 
     def call_later(self, delay, func, *args):
         """ Another event loop proxy, see `call_soon`. """
         def f():
             func(*args)
             self.conn.flush()
-        return self._eventloop.call_later(delay, f)
+        return self._event_loop.call_later(delay, f)
 
     def run_in_executor(self, func, *args):
         """ A wrapper for running a function in the event loop's default
         executor. """
-        return self._eventloop.run_in_executor(None, func, *args)
+        return self._event_loop.run_in_executor(None, func, *args)
```

Comment:
I'm good with the renaming but please put it in another comment, it hurts the readability of the diff.